storage/innobase/fts/fts0que.cc

```diff
@@ -4096,7 +4096,7 @@ fts_query(
 	/* For binary collations, a case sensitive search is
 	performed. Hence don't convert to lower case. */
 	if (my_binary_compare(charset)) {
-	memcpy(lc_query_str, query_str, query_len);
+		memcpy(lc_query_str, query_str, query_len);
```

Comment:
Is this our code? If not, then just drop (OK to include in eventual OCA submission)